Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN apt-get update && apt-get install libelf1 -y
 
 COPY . /src
 WORKDIR /src
-RUN yarn install --frozen-lockfile --ignore-engines --prefer-offline
+RUN yarn install --frozen-lockfile --ignore-engines --prefer-offline && node node_modules/webpack/bin/webpack.js --config  webpack.config.prod.js --bail
```

Comment:
I believe that this already gets run as part of the install because of [this line](https://github.com/mitodl/odl-video-service/blob/master/package.json#L124)

Reply:
Roger that. :+1: